sdk/monitor/azure-monitor-opentelemetry-autoconfigure/pom.xml

```diff
@@ -96,11 +96,6 @@
       <artifactId>opentelemetry-semconv-incubating</artifactId>
       <version>1.26.0-alpha</version> <!-- {x-version-update;io.opentelemetry.semconv:opentelemetry-semconv-incubating;external_dependency} -->
```

Comment:
we should remove this (existing) alpha dependency as well

for stable semconv we can use the stable semconv artifact: https://github.com/open-telemetry/semantic-conventions-java#published-releases

and for unstable semconv let's inline the constants

Reply:
(this can be done in a separate PR)